app/Notifications/TenantApprovalRequest.php

```
@@ -0,0 +1,76 @@
+            ->line("**Solicitado por:** {$this->requestingUser->name} ({$this->requestingUser->email})")
+            ->line("**Fecha:** " . now()->format('d/m/Y H:i'))
+            ->action('Revisar Solicitud', url('/tenants/' . $this->tenant->id))
+            ->line('Por favor revisa la solicitud y apruébala o recházala según corresponda.');
+    }
+
+    /**
+     * Get the array representation of the notification.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(object $notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (Unused Code Rules)

(UnusedFormalParameter)